style.css

```diff
@@ -87,3 +92,347 @@ img:hover {
   color: #3c3a39;
   cursor: pointer;
 }
+
+.work-section-heading {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 0 24px 12px 24px;
+  width: 327px;
+}
+
+.work-section-heading h2 {
+  font-family: 'Crete Round', sans-serif;
+  padding: 50px 0 38px 0;
+  text-align: center;
+  font-weight: 500;
+  width: 327px;
+  font-size: 38px;
+  line-height: 48px;
+  color: #1c1a19;
+}
```

Comment:
- [ ] Please avoid using a fixed value in the `width` property, the intention of the whole exercise is to create a responsive design, adding a fixed width will make things harder in the future. Please delete the width property or in any case is necessary, replace the pixel value for a  percentage value